multiqc/modules/picard/picard.py

```diff
@@ -157,3 +157,25 @@ def multiply_hundred(self, val):
         except ValueError:
             pass
         return val
+
+
+def strtobool(val):
```

Comment:
Better to put this in the MultiQC utils, rather than having it in a module?

Reply:
It think it makes sense to move it if/when it's going to be used by other modules

Reply:
Though it will actually fix the test, so doing it now 😅